lib/screens/explore_screens/unplant_seeds/unplant_seeds_screen.dart

```diff
@@ -51,17 +52,16 @@ class UnplantSeedsScreen extends StatelessWidget {
                 },
               );
             }
-            //Will add correct Dialog on next pr
             if (pageCommand is ShowClaimSeedsSuccess) {
               showDialog<void>(
                 context: context,
                 barrierDismissible: false,
                 builder: (_) {
                   return BlocProvider.value(
                     value: BlocProvider.of<UnplantSeedsBloc>(context),
```

Comment:
I think the bloc is not used in `ClaimSeedsSuccessDialog`

Reply:
you are right, I remove it